src/main/webapp/WEB-INF/keycloak.json

```diff
@@ -6,5 +6,6 @@
   "public-client": true,
   "disable-trust-manager": "false",
   "truststore": "${jboss.server.config.dir}/cacerts.jks",
-  "truststore-password": "geheim"
+  "truststore-password": "geheim",
+  "disabled": true
```

Comment:
After merging this pr https://github.com/Lumeer/web-ui/pull/82 we don't have to change keycloak.json anymore. Just set LUMER_ENV to development.